### DISCUSSION TITLE
Could not poll the database

### DISCUSSION THREAD

**Discussion by JaredScar** [firefly-iii/firefly-iii#6131] [Firefly III] [answered]
I have my mysql server running in it's own container named `mysql-server`.

I am running Firefly III via it's docker image. The docker image is `fireflyiii/core:latest`.

Upon accessing the web server, I get the following error
![image](https://user-images.githubusercontent.com/8027457/172013362-b45a084f-c365-40a0-a1ac-f9787c8c158f.png)

**Comment by JC5** [comment, marked as answer]
https://github.com/orgs/firefly-iii/discussions?discussions_q=poll
